src/screens/Watch/model.js

```diff
@@ -353,7 +368,8 @@ const WatchModel = {
             let liveMode = state.liveMode
             if(state.liveMode === 1) {
                 liveMode = payload < state.duration - 60 ? 2 : 1
```

Comment:
Magic constants: Why 60? Is this from testing? Could it change in the future?
Magic Constants: 1 and 2 should be labels